includes/Meta.php

```diff
@@ -56,7 +56,7 @@ public function register_post_meta(): void {
 									'required' => true,
 								],
 								'uri'      => [
-									'type'     => 'uri',
+									'type'     => 'string',
```

Comment:
```suggestion
									'type'     => 'string',
									'format' => 'uri',
```

@imath Thanks! That all looks correct, but I think I figured out the bug: the type should indeed be set to `string`, and an extra key named `format` should be added and set to `uri`.

This should also happen for the `at-uri` option below, but GH won't let me suggest that change here.

You can update the PR with that or I can do it myself, up to you! Thank you for reporting this.

Reply:
hi @danielpost 

I've just updated the PR accordingly 👍 

Here's the commit: https://github.com/posty-studio/autoblue/pull/7/commits/59fc297c2d48f2243f68261b9ea42f100a2e31e8